app/controllers/catalog_controller.rb

```diff
@@ -15,6 +15,30 @@ def guest_uid_authentication_key(key)
   end
 
   configure_blacklight do |config|
+    config.citeproc = {
+      #bibtex_field: 'bibtex_s' # Optional. If you are already indexing BibTeX into solr, this will be the preferred way to get bibliographic data
+      fields: {
+        author: 'creator_tesim',
+        edition: 'edition_tesim',
+        institution: 'institution_tesim',
+        organization: 'holding_repository_tesim',
+        publisher: 'publisher_tesim',
+        series: ['member_of_collections_ssim', 'series_title_tesim'],
```

Comment:
From this file, it looks like we're configuring some fields that I'm not seeing in the related test, such as `member_of_collections_ssim` and `content_genres_tesim`.

Would you please add examples in your tests that use these fields?

Reply:
The reason I included the `series` field is from a conversation I had with Emily P. I asked her to match the available fields from citation-styles to our solr fields. She matched `series` to those fields there, but after testing a document I knew had the `series` field and it not showing in the citation, I dug into the styles code. Turns out `series` isn't used in any of them. I left it there just in case we expanded our citation style list (future proof).

The `type` is a big one in the styles, but our values in that field have to match perfectly to what `citation-styles` expects in order for media-specific styling to be exact. Unfortunately, they don't. In other words, 99% of the time, the citation will be the default styling. Emily is aware of that, and I think she and Nik agreed to adding the disclaimer above the list to warn students of that.